packages/segmented-control/src/SegmentedControl/styles.ts

```diff
@@ -13,14 +13,23 @@ export const wrapperStyle = css`
   font-family: ${fontFamilies.default};
 `;
 
+const labelBaseStyles = css`
+  white-space: nowrap;
+`;
+
 export const labelThemeStyle: Record<Theme, string> = {
-  [Theme.Light]: css`
-    letter-spacing: 1.4px;
-    color: ${palette.gray.dark1};
-  `,
-  [Theme.Dark]: css`
-    color: ${palette.gray.light1};
-  `,
+  [Theme.Light]: cx(
+    labelBaseStyles,
+    css`
+      color: ${palette.gray.dark1};
+    `,
+  ),
+  [Theme.Dark]: cx(
+    labelBaseStyles,
+    css`
+      color: ${palette.gray.light1};
+    `,
+  ),
```

Comment:
Small nit - based on the code styles discussions we've been having, I think `labelBaseStyles` should be passed directly to the `cx` call in the render function than repeatedly defined in the theme styles.